tests/conftest.py

```diff
@@ -184,6 +187,11 @@ def celery_config():
     return {}
 
 
+def _(x):
+    """Identity function for string extraction."""
+    return x
```

Comment:
do we need this?
AFAIK, this is usually just a placeholder for `from flask_babelex import lazy_gettext as _` when translations are planned, but not yet done... but i don't think translations will be relevant here in the future?

Reply:
This was copied from what is used in config.py (https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/config.py#L19) because the `_()` calls are used in the config we copied e.g., https://github.com/inveniosoftware/invenio-rdm-records/pull/867/files#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R256  . Just keeping that as is.